docs/source/style_guide.rst

```diff
@@ -23,12 +23,12 @@ Examples
 ^^^^^^^^
 
 Good:
- * paasta mark-for-deployment => Goes to stdout. (But also scribe)
+ * paasta mark-for-Deployment => Goes to stdout. (But also scribe)
 Bad:
- * paasta mark-for-deployment => Going to scribe only, which is surprising.
+ * paasta mark-for-Deployment => Going to scribe only, which is surprising.
 
 Good:
- * paasta itest => Sends event-level detail to stdout
+ * PaaSTA itest => Sends event-level detail to stdout
```

Comment:
this file has a couple of find-replace replacements that should be undone (the commands are all lowercase :p)